legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/test/full/functions/in/TestPlanExecutionForIn.java

```diff
@@ -236,6 +245,84 @@ public void testInExecutionWithStringList() throws JsonProcessingException
         Assert.assertEquals(expectedResListWithAmpersand, RelationalResultToJsonDefaultSerializer.removeComment(executePlan(plan, paramWithAmpersand)));
     }
 
+    
+    @Test
+    public void TestFreeMarkerProcessingForTempTable() throws Exception
```

Comment:
What do both of these tests actually test? Also, test name should not start with a capital letter.

Reply:
"testFreemarkertestFreeMarkerProcessingForCombinationalPlaceholdersInPlan" is a end-to-end test (plan generation to execution) that tests 3 freemarker placeholders, based off the three query parameters added to this service. The objective was to test combination of 3 placeholders, with each needing different amount of recusive calls to process. 

testFreeMarkerProcessingForSingularPlaceholderinTempTablePlan is testing freemarker processing when we have to create temp tables if user input exceeds threshold. 


I have renamed these functions so that the intention is more clear when you read the testname. But they are there to reflect some real, complex situations that our freemarker logic would be required to process correctly.